GooglePlayInstant/Editor/ApkSigner.cs

```diff
@@ -107,7 +111,10 @@ public static bool Sign(string apkPath)
 
             var arguments = string.Format(
                 "-jar {0} sign --ks {1} --ks-key-alias {2} --pass-encoding utf-8 {3}",
-                GetApkSignerJarPath(), keystoreName, keyaliasName, apkPath);
+                CommandLine.QuotePathIfNecessary(GetApkSignerJarPath()),
```

Comment:
minor nit, and I don't feel too strongly about this: apksigner's CLI syntax is a little obtuse sometimes, the verify above is fine, but this one is a little hard to grok, let's add a comment here to explain what the different flags do

Reply:
Good suggestion. Done.